src/utilities/focus.ts

```diff
@@ -130,6 +130,7 @@ export function isElementTabbable(element: HTMLElement): boolean {
     (element.tagName === 'A' ||
       (element.tagName === 'BUTTON' && !(element as HTMLButtonElement).disabled) ||
```

Comment:
I guess I was thinking more of the C# style 'as' operator where it actually checks they object type and returns null if it doesn't match, where this just swaps the interface.